### PR TITLE
Add unit test for Invoke pattern

### PR DESCRIPTION
**Pull request by LeafShi1**
Related #10244

## Proposed changes

- Add unit tests for Controls that support InvokePattern

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/winforms/pull/11190)

### CODE REVIEW

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AccessibleObjects/ToolStripMenuItem.ToolStripMenuItemAccessibleObjectTests.cs [hunk 1]
```diff
@@ -18,6 +18,25 @@ public void ToolStripMenuItemAccessibleObject_Ctor_Default()
...
     }
 
+    [WinFormsFact]
+    public void ToolStripMenuItemAccessibleObject_Supports_InvokePattern()
+    {
+        int callCount = 0;
+        EventHandler handler = (sender, e) =>
+        {
+            // This action is temporarily commented due to issue #10244
+            // MessageBox.Show("...");
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/AccessibleObjects/ToolStripMenuItem.ToolStripMenuItemAccessibleObjectTests.cs:28]: How would you close this messagebox? You need the same test form as you have in the button test code sample that you show in the issue description

**Comment by Tanya-Solyanik** [comment]
The change looks good, but tests are failing :(

**Review by Tanya-Solyanik — COMMENTED** [review]
This is strange. We actually don't run tests on ARM, so Any and x86+x64 should be equivalent.

**Review by Tanya-Solyanik — APPROVED** [review]

**Comment by LeafShi1** [comment]
> This is strange. We actually don't run tests on ARM, so Any and x86+x64 should be equivalent.

The log of PR test failure does not have detailed reasons, so it is impossible to determine where the problem occurred. I just saw that only the X86 and X64 Flags were judged in the `SkipOnArchitectureDiscoverer` class, and TestArchitectures.Any has never been used, so replace Any with X86+X64

**Pull request merged**